### ISSUE TITLE
Pimcore Update from 6.0.5 to >6.6.4

### ISSUE THREAD

**Issue by elements-team** [pimcore/pimcore#6648]
# Bug Report
As per https://github.com/pimcore/pimcore/pull/6146 one cannot login to Pimcore anymore.
According to Chrome's Dev-Tools, there is an error when `/js/routing` is called.

This also means that any sub-sequent route fails too, which leads to being stuck with the loading spinner.

The root cause is
`The controller for URI "/js/routing" is not callable: Class "fos_js_routing.controller" does not exist.`

## Workaround
Downgrade to Pimcore 6.6.4.

**Comment by elements-team** [comment]
Another workaround is adding the following code to your `app/config/routing.yml`:

```yml
_fos_js_routing:
    resource: "@FOSJsRoutingBundle/Resources/config/routing/routing.xml"
```

**Comment by weisswurstkanone** [comment]
@dpfaffenbauer could you please share your opinion :-) thx! 
Symfony version was 3.4

**Comment by dpfaffenbauer** [comment]
https://github.com/dpfaffenbauer/pimcore/blob/69009d0d4a5b4a64d9ef76341421dc8478c2eea0/bundles/CoreBundle/Resources/config/routing.yml#L39

Fos Routing comes with 2 routing files, one for SF3 and one for SF4

**Comment by brusch** [comment]
Hmm, so how we can we make a generic fix for that 🤔

**Comment by blankse** [comment]
I created an PR for this. Can you test it?

**Comment by dvesh3** [comment]
thanks a lot @blankse... it looks good 👍 we'll test and merge it.

**Comment by guidoheckmann** [comment]
My workaround is removing name attributes from route definition in vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Reports/QrcodeController.php

**Comment by dvesh3** [comment]
resolved by #6685
